gensim/models/flsamodel.py

```diff
@@ -623,7 +675,21 @@ def _create_partition_matrix(
             method='fcm',
         ):
         """
-        Cluster the projected data.
+        Use fuzzy clustering on the projected data.
+
+        The three clustering methods are:
+            1. fcm (Fuzzy C-Means)
+                (http://bitly.ws/zjs6)
```

Comment:
Please use full canonical URLs. Avoid shorteners and any obfuscation (security risk).